data/pycon-au-2011/videos/a-python-on-the-couch.json

```diff
@@ -1,31 +1,31 @@
 {
-  "id": 995, 
-  "category": "PyCon AU 2011", 
-  "slug": "a-python-on-the-couch", 
-  "title": "A Python on the Couch", 
-  "summary": "CouchDB &nbsp_place_holder;(http://couchdb.apache.org/) is an open source,\ndocument-oriented NoSQL Database Management Server.It supports queries via\nviews using MapReduce, and replication. The talk will give an overview of\nCouchDB followed by how to access and manipulate using Python. There are a\nnumber of python libraries for accessing couchdb and these will be quickly\ndiscussed followed by &nbsp_place_holder;how to use one of these libs with a\nPython web framework.\n\n", 
-  "description": "", 
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "Creative Commons Attribution license", 
-  "thumbnail_url": "http://i.ytimg.com/vi/RNXjflpOs88/hqdefault.jpg", 
-  "duration": null, 
+  "id": 995,
+  "category": "PyCon AU 2011",
+  "slug": "a-python-on-the-couch",
+  "title": "A Python on the Couch",
+  "summary": "CouchDB &nbsp\\_place\\_holder;(http://couchdb.apache.org/) is an open\nsource, document-oriented NoSQL Database Management Server.It supports\nqueries via views using MapReduce, and replication. The talk will give\nan overview of CouchDB followed by how to access and manipulate using\nPython. There are a number of python libraries for accessing couchdb and\nthese will be quickly discussed followed by &nbsp\\_place\\_holder;how to\nuse one of these libs with a Python web framework.\n",
```

Comment:
`CouchDB &nbsp\\_place\\_holder;(http://couchdb.apache.org/)` looks weird, but it was also in the original Markdown description.
The source page has some character/encoding issue: http://pycon-au.org/2011/conference/schedule/event/10/index.html

> CouchDB Â (http://couchdb.apache.org/) is an open source,